openedx_external_enrollments/tests/external_enrollments/tests_pathstream_external_enrollments.py

```diff
@@ -0,0 +1,97 @@
+"""Tests PathstreamExternalEnrollment class file"""
+import logging
+
+from django.test import TestCase
+from mock import patch
+from testfixtures import LogCapture
+
+from openedx_external_enrollments.external_enrollments.pathstream_external_enrollment import (
+    PathstreamExternalEnrollment,
+)
+from openedx_external_enrollments.models import ExternalEnrollment
+from openedx_external_enrollments.tests.tests_backends import CourseOverview
+
+module = 'openedx_external_enrollments.external_enrollments.pathstream_external_enrollment'
+
+
+class PathstreamExternalEnrollmentTest(TestCase):
+    """Test class for PathstreamExternalEnrollment."""
+
+    def setUp(self):
+        """Set test database."""
+        self.base = PathstreamExternalEnrollment()
+
+    def test_str(self):
+        """
+        PathstreamExternalEnrollment overrides the __str__ method,
+        this test that the method __str__ returns the right value.
+        """
+        self.assertEqual(
+            'pathstream',
+            self.base.__str__(),
+        )
```

Comment:
Please update the order of the assertion throughout the file, we are working with the following order: 
`assertion(actual, expected)`
```suggestion
        self.assertEqual(self.base.__str__(), 'pathstream')
```